src/resources/postcss/tribe-common-admin.pcss

```diff
@@ -10,7 +10,8 @@ input:out-of-range {
 }
 
 .clearfix {
-	zoom: 1; /* For IE */
+	zoom: 1;
+	/* For IE */
```

Comment:
auto-linting change, move this comment up before the zoom

Reply:
Should be fixed